fitters/src/DAF.cc

```diff
@@ -130,8 +150,8 @@ void DAF::processTrackWithRep(Track* tr, const AbsTrackRep* rep, bool resortHits
     bool converged(false);
     try{
       converged = calcWeights(tr, rep, betas_.at(iBeta));
-      if (!converged && iBeta >= minIterations_-1 &&
-          status->getBackwardPVal() != 0 && fabs(lastPval - status->getBackwardPVal()) < this->deltaPval_) {
+      if (!converged && iBeta >= minIterForPval_-1 &&
```

Comment:
This fixes the minIterations to 5 for all cases where the old `setAnnealingScheme() ` function is used. This can cause some unexpected behavior since it is not depending on nSteps anymore.

Reply:
implemented the link to nSteps as you suggested